### PR TITLE
Moved patient_id hashing to plug

### PR DESCRIPTION
**Pull request by BatrakM**

### CODE REVIEW

File: apps/core/lib/core/patients/patients.ex [hunk 1]
```diff
@@ -170,15 +172,18 @@ defmodule Core.Patients do
   end
 
   def consume_cancel_package(%PackageCancelJob{patient_id: patient_id, user_id: user_id} = job) do
+    patient_id_hash = get_pk_hash(patient_id)
```
[the remark this answers is not in the document]
Comment (progsmile) [reply apps/core/lib/core/patients/patients.ex]: Yep, patient_id is already hash

**Comment by BatrakM** [comment]
Now I add patient_id_hash to params along with patient_it so we can use the one we need

File: apps/api/lib/api_web/controllers/episode_controller.ex [hunk 2]
```diff
@@ -10,13 +10,13 @@ defmodule Api.Web.EpisodeController do
...
 
-  def index(conn, %{"patient_id" => patient_id} = params) do
+  def index(conn, %{"patient_id_hash" => patient_id_hash} = params) do
     with %Page{entries: entries} = paging <- Episodes.list(params) do
       render(
         conn,
         "index.json",
         paging: %{paging | entries: Enum.map(entries, &Episode.create/1)},
-        patient_id: patient_id
+        patient_id: patient_id_hash
```
Comment (progsmile) [inline apps/api/lib/api_web/controllers/episode_controller.ex]: Are you sure `patient_id_hash` should be return for client?

**Comment by BatrakM** [comment]
I removed patient_id from response

**Pull request merged**